sdk/schemaregistry/schema-registry/recordings/node/schemaregistryclient/recording_allows_schema_names_with_dots_in_them.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Yaaay, this looks super clean
![ApplauseApplaudGIF](https://user-images.githubusercontent.com/6074665/148165522-53a9d1e7-a0a4-445c-9c19-73666e3f820f.gif)